cfgov/ask_cfpb/views.py

```diff
@@ -131,3 +133,23 @@ def ask_autocomplete(request, language='en'):
                 'url': result.url}
                for result in sqs[:20]]
     return JsonResponse(results, safe=False)
+
+
+def redirect_ask_search(request):
+    # Get selected facets
+    selected_facets = request.GET.get('selected_facets')
+    if selected_facets is None:
+        raise Http404
+
+    # Get the category
+    category_match = re.match(r'category_exact:(?P<category>[^&]+)',
+                              selected_facets)
+    if category_match is None:
+        raise Http404
+
+    category = category_match.groupdict().get('category')
+    if category is None:
+        raise Http404
+
+    return redirect('/ask-cfpb/category-{category}'.format(
+                    category=category), permanent=True)
```

Comment:
@higs4281 if you have a chance could you review this? Is `selected_facets:category_exact` the only thing we're interested in with old search?